data/data/ovirt/affinity_group/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "ovirt_affinity_group" {
```

Comment:
Nitpick: this could be named `ovirt_affinity_group_count`

Reply:
Done